<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lancar_entrada_no_estoque</class>
 <widget class="QDialog" name="lancar_entrada_no_estoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="estoque_horizontalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Nome da localização no estoque:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_local_estoque"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_local_estoque">
          <property name="text">
           <string>Consultar</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_local_estoque"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="lancar_horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_add_produto"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Quantidade</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_add_qtde_produto"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget_lancar_produtos"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>teste duplicado</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_lancar_estoque">
        <property name="text">
         <string>Lançar no Estoque</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
